src/GenSymIO.chpl

```diff
@@ -279,12 +279,13 @@ module GenSymIO {
             repMsg = "[";  // Manual json building Chapel <= 1.24.1
             var first = true;
             for i in items {
+                i = i.replace(Q, ESCAPED_QUOTES, -1);
```

Comment:
i think `count = -1`  by default for `replace`

Reply:
It is the default, I decided to be explicit in case the default changes.